Makefile

```diff
@@ -41,7 +41,7 @@ UNIT_TEST_PACKAGES := $(shell go list ./...  | grep -v /vendor | grep -v /e2e |
 
 statusgo: ##@build Build status-go as statusd server
 	go build -i -o $(GOBIN)/statusd -v $(shell build/testnet-flags.sh) ./cmd/statusd
-	@echo "\nCompilation done.\nRun \"build/bin/statusd -h\" to view available commands."
```

Comment:
Couldn't we replace it with:
```
@echo "Compilation done."
@echo "Run \"build/bin/statusd -h\" to view available commands."
```
to preserve newlines properly, not as literals? 

My problem with `printf` is that we use `echo` everywhere and it would be nice to keep it consistent.

Reply:
I thought about using `echo` for consistency. Note that there was an extra '\n' in the beginning of the sentence, so we would need

```
@echo
@echo "Compilation done."
@echo "Run \"build/bin/statusd -h\" to view available commands."
```

to keep the same output string. I then though it would be too noisy and went with `printf`. Would you prefer the longer (but more consistent) approach using echo?

Reply:
I think we can skip the first empty `@echo` and the output should be still pretty clear, right? Yeah, I think it would be better to stay with `echo` and put two calls instead of one.

Reply:
@athos-ribeiro Please would you take care for @adambabik latest comment? Thanks.